app/src/main/res/values/styles.xml

```diff
@@ -24,6 +24,10 @@
 		<item name="colorAccent">@color/accent</item>
 	</style>
 
+	<style name="AlertDialogTheme" parent="Theme.AppCompat.Dialog.Alert">
```

Comment:
I thought about these approaches but in either case if the app theme applies to the AlertDialog then the dialog takes up the full screen and doesn't even look like a dialog:

![full screen dialog](https://cloud.githubusercontent.com/assets/5909572/10745268/5e6da222-7bfd-11e5-9b4b-aa5d57e4f91a.png)